front/app/api/app_configuration/types.ts

```diff
@@ -150,8 +150,6 @@ export interface IAppConfigurationSettings {
   workshops?: AppConfigurationFeature;
   smart_groups?: AppConfigurationFeature;
   manual_emailing?: AppConfigurationFeature;
-  insights_manual_flow?: AppConfigurationFeature;
-  insights_nlp_flow?: AppConfigurationFeature;
```

Comment:
Would need to be removed in BE but I'm sure you already thought of that

Reply:
Yes, it's a separate PR :)